### ISSUE TITLE
haddock failure

### ISSUE THREAD

**Issue by bergmark** [camfort/fortran-src#29]
```
src/Language/Fortran/Analysis/Types.hs:87:3: error:
    parse error on input ‘-- | any (not . isIxSingle) (aStrip ixAList) = recordCType CTArray (varName v)  -- it's an array (or a string?) FIXME’
```

ping @phadej

**Comment by bergmark** [comment]
Woops sorry @phadej, I'm going to ping you on another ticket.

**Comment by dorchard** [comment]
Thanks for spotting this. I have fixed it with commit ab4a3f22aae2d91ceadccea2c167d3cd38cc60b3 and update hackage.
